NodeFirefoxDebug/entry_point.sh

```diff
@@ -3,6 +3,7 @@
 # IMPORTANT: Change this file only in directory NodeDebug!
 
 source /opt/bin/functions.sh
+/opt/bin/generate_config > /opt/selenium/config.json
```

Comment:
I think they should both exist.
The RUN directive would create a default configuration file, and the generate_config from the `entry_point.sh` will allow to customize environment variables.

This is actually done for the hub : [Hub/Dockerfile#L39](https://github.com/SeleniumHQ/docker-selenium/blob/master/Hub/Dockerfile#L39) & [Hub/entry_point.sh#L6](https://github.com/SeleniumHQ/docker-selenium/blob/master/Hub/entry_point.sh#L6)